certbot/tests/cert_manager_test.py

```diff
@@ -277,13 +277,12 @@ class SearchLineagesTest(BaseCertManagerTest):
     @mock.patch('certbot.storage.renewal_conf_files')
     @mock.patch('certbot.storage.RenewableCert')
     def test_cert_storage_error(self, mock_renewable_cert, mock_renewal_conf_files,
-        mock_make_or_verify_dir):
+                                mock_make_or_verify_dir):
```

Comment:
It looks like this file only has lint-like changes; was something substantive changed here that I'm missing? I'm fine with the lint changes, but I do want to be sure I'm not missing something here.

Reply:
No, nothing more. My PyCharm was angry while working on `cert_manager` module.